### ISSUE TITLE
Determine how sample size / number of filters per layer / number of layers / other parameters affect model performance

### ISSUE THREAD

**Issue by alexdauenhauer** [tyu0912/MIDS_W207_facial_recognition#22]
Training time is a huge consideration here as well as predictive accuracy and this trade off will be key to building a good model

**Comment by alexdauenhauer** [comment]
https://machinelearningmastery.com/grid-search-hyperparameters-deep-learning-models-python-keras/

might be helpful

**Comment by alexdauenhauer** [comment]
Got `GridSearchCV` working, now need to run it forever to find the best parameters
